SpatialGDK/Source/SpatialGDK/Private/Utils/ComponentReader.cpp

```diff
@@ -154,6 +152,7 @@ void ComponentReader::ApplySchemaObject(Schema_Object* ComponentObject, UObject*
 				{
 					// Downgrade role from AutonomousProxy to SimulatedProxy if we aren't authoritative over
 					// the client RPCs component.
+					bool bAutonomousProxy = Channel->IsClientAutonomousProxy();
```

Comment:
This will get called for every field, compiler could potentially optimize, but might be worth just moving it back outside the loop.